lib/config/index.js

```diff
@@ -1580,6 +1582,19 @@ Config.prototype.getAggregatorConfig = function getAggregatorConfig(method) {
   }
 }
 
+Config.prototype._warnDeprecations = function _warnDeprecations() {
```

Comment:
i can't seem to comment inline but the two linting warnings above didn't get auto fixed, mind manually fixing?

Reply:
yeah, code areas that were not updated at all don't allow for inline comments.

happy to take a quick look at the warnings.

Reply:
@bizob2828 pushed up a commit to clean up the extra things you commented on.